isso/views/comments.py

```diff
@@ -31,21 +32,12 @@
 from isso.views import requires
 
 
-# from Django apparently, looks good to me *duck*
-__url_re = re.compile(
-    r'^'
-    r'(https?://)?'
-    # domain...
-    r'(?:(?:[A-Z0-9](?:[A-Z0-9-]{0,61}[A-Z0-9])?\.)+(?:[A-Z]{2,6}\.?|[A-Z0-9-]{2,}\.?)|'
-    r'localhost|'  # localhost...
-    r'\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})'  # ...or ip
-    r'(?::\d+)?'  # optional port
-    r'(?:/?|[/?]\S+)'
-    r'$', re.IGNORECASE)
-
-
 def isurl(text):
-    return __url_re.match(text) is not None
+    text = normalize(text)
+    # urlparse does not like port numbers in URLs
+    text = re.sub(r':\d+', '', text)
```

Comment:
This comment seems odd to me - urlparse handles port numbers in URLs fine, so there must be something else going on?

Reply:
My bad, the reason for removing the port was not urlparse, it is `validators.domain()` which does not accept `domain:port`. I guess I could clean this up by using `hostname` instead of `netloc`, but if the complete URL is supposed to be validated these lines are most probably not staying anyway.

Reply:
I did add my test case in [test_comments.py](https://github.com/posativ/isso/blob/90019450483c601c1f3dee3de1e973a41679e4d9/isso/tests/test_comments.py#L172).